app/assets/stylesheets/create_court.scss

```diff
@@ -0,0 +1,53 @@
+.court-name-field {
+  width: 260px;
+}
+
+.court-street-address-field {
+  width: 260px;
+}
+
+.court-city-field {
+  width: 180px;
+  display: inline-block;
+  margin-right: 10px;
+}
+
+.court-state-field {
+  width: 50px;
+  display: inline-block;
+  margin-right: 10px;
+}
+
+.court-zip-field {
+  width: 80px;
+  display: inline-block;
+}
+
+.court-setting-field {
+  width: 180px;
+  display: inline-block;
+  margin-right: 10px;
+}
+
+.court-hoop-count-field {
+  width: 130px;
+  display: inline-block;
+}
+
+.court-hoop-count-input {
+  width: 40px;
+}
+
+.court-hours-input {
+  width: 180px;
+}
+
+#createMap {
```

Comment:
Avoid using id selectors<br>Selector `createMap` should be written in lowercase with hyphens